README.md

```diff
@@ -34,6 +34,7 @@ __NOTE__: It's early days so please be gentle when reporting issues :) As author
 
         post 'creates a new blog' do
           consumes 'application/json'
+          operation_description 'Detailed implementation notes for the create a new blog API endpoint'
```

Comment:
That makes sense. But description conflicts with ExampleGroup#description hence let me change it to implementation_notes which is how they show up in the swagger ui.